Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebViewSuspendAllMediaPlayback.mm

```diff
@@ -26,6 +26,7 @@
 #import "config.h"
 
 #import "PlatformUtilities.h"
+#import "Test.h"
```

Comment:
This change seems unrelated to the rest of your patch, is it necessary?

Reply:
It was needed to fix the build with UnifiedSources rearranging with the new test being added.